src/System.Text.Json/tests/Serialization/TestClasses.cs

```diff
@@ -1784,10 +1784,10 @@ public void Verify()
 
     public class ClassWithUnicodeProperty
     {
-        public int Aѧ { get; set; }
+        public int A\u0467 { get; set; }
```

Comment:
I had no idea that was legal C#.  Nifty.

Reply:
It was intentionally added to exercise our JSON escaping behavior when serializing :)

We should add one where all the characters (in a long name) are non-ascii.